### PR TITLE
Update bug-report.yaml for 1.4 stable

### PR DESCRIPTION
**Pull request by duncanmacmichael**
A microsoft employee must use /azp run to validate using the pipelines below.

WARNING:
Comments made by azure-pipelines bot maybe inaccurate. 
Please see pipeline link to verify that the build is being ran.

For status checks on the develop branch, please use TransportPackage-Foundation-PR
(https://microsoft.visualstudio.com/ProjectReunion/_build?definitionId=81063&_a=summary)
and run the build against your PR branch with the default parameters.

For status checks on the main branch, please use microsoft.ProjectReunion
(https://dev.azure.com/ms/ProjectReunion/_build?definitionId=391&_a=summary)
and run the build against your PR branch with the default parameters.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ body:
       label: NuGet package version
       description: Specify the version of Windows App SDK (or Project Reunion) you're using.
       options:
-        - "Windows App SDK 1.3.3: 1.3.230724000"
+        - "Windows App SDK 1.4.0: 1.4.230822000"
```
Comment (bpulliam) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:38]: you can remove preview 2

Comment (duncanmacmichael) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:38]: And Experimental 1?

Comment (bpulliam) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:38]: Experimental 1 has some APIs that aren't present in the stable release.  It should stay.

Comment (duncanmacmichael) [reply .github/ISSUE_TEMPLATE/bug-report.yaml:38]: Sounds good, thanks!

**Review by bpulliam — APPROVED** [review]

**Pull request merged**